src/test/java/com/wikia/webdriver/elements/mercury/pages/login/JoinPage.java

```diff
@@ -0,0 +1,17 @@
+package com.wikia.webdriver.elements.mercury.pages.login;
+
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.elements.mercury.components.TopBar;
+import com.wikia.webdriver.elements.mercury.components.login.JoinButtons;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import lombok.Getter;
+import org.openqa.selenium.WebElement;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.WebElement'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)